includes/Core/Admin/Pointers.php

```diff
@@ -140,10 +145,24 @@ private function print_pointer_script( $pointer ) {
 							);
 						},
 						pointerClass: "%s",
-						%s
+						buttons: function( event, container ) {
+							return jQuery("<div></div>");
+						}
 					};
 
-					jQuery( "#%s" ).pointer( options ).pointer( "open" );
+					jQuery( "#%6$s" ).pointer( options ).pointer( "open" );
+
+					jQuery("body").on("click", ".googlesitekit-pointer-cta--dismiss", function() {
+						jQuery( "#%6$s" ).pointer("close");
+					});
+					jQuery("body").on("click", ".googlesitekit-pointer-cta", function(event) {
+						// We must prevent default link navigation in order to dismiss the pointer first
+						// to prevent it showing again.
+						event.preventDefault();
+						jQuery( "#%6$s" ).pointer("close");
+						window.location = jQuery(this).attr("href");
```

Comment:
Do we need to prevent the default? It seems odd that we'd prevent the normal behavior to do it manually one line later.

Reply:
We do because in the updates you have made `container.element.pointer( 'close' );` isn't called before navigation which means the pointer may show again after clicked.